_posts/2020-06-11-trusted-web-activities.md

```diff
@@ -0,0 +1,138 @@
+Using the [Asset Links Tool](https://play.google.com/store/apps/details?id=dev.conn.assetlinkstool&hl=en) from the Play Store is the easiest way to generate the asset link. Install it on the same device that you installed the APK and then open it.
+
+It will ask you for the qualified package name of your TWA (e.g. com.mytwa). Upon selecting your package, you will see the signature and Digital Asset Link.
+
+![twa_asset_link](/img/twa_asset_link.png)
+
+Copy the asset link into a file named `assetlink.json`, and host it under the `/.well-known` folder of your site. Once that's deployed, restart the app on your phone and the header bar should disappear, resulting in a glorious full screen web app!
+
+### Publishing to the Play Store
+
+If we're happy with the testing of the app, we're ready to publish it to the Google Play Store.
+
+If you don't already have a developer account for the Google Play Console, you will need to [create one](https://play.google.com/apps/publish/signup/). At the time of writing this involves a one-time $25 USD fee.
+
+![play_store_sign_up](/img/play_store_sign_up.png)
+
+Once you're signed into the Play Console, it's time to [upload the app](https://support.google.com/googleplay/android-developer/answer/113469?hl=en). Navigate to *All applications* and then *Create application*. Enter a name for your application and the language of choice.
+
+Proceed to creating your app's [store listing](https://support.google.com/googleplay/android-developer/answer/113469#store_listing), enter a description of what it does, some screenshots and other marketing material. Take the content rating questionnaire and set up pricing and distribution.
+
+Under *Release management*, upload the generated APK to the *artifact library*. I recommend using the app signing by Google Play, as this is a managed signing service and voids the need to manage the signing yourself.
+
+To submit your app for review, you need to [roll out a release](https://support.google.com/googleplay/android-developer/answer/7159011#rollout) under *App releases*. Walk through the steps of the wizard, select *Review*, *Review and roll out release* and then *Confirm* the rollout.
+
+At this point it may take a few or several days before the app is published to the store. Later updates are not required, as the TWA will always serve the latest version of your app from the web.
+
+Once your app is published, you may need to repeat the steps in the asset link section, as the published APK will have a different signature. After that, your app will be ready to be installed by your users.
+
+
+### Conclusion
+
+We've learned what a TWA is, how we can create one with PWABuilder and how to publish it to the Play Store.
+
+This is a big step forward for publication of web apps to the native app stores of the popular mobile platforms, as it eliminates the complexity of creating a native wrapper. I'm hoping that this sets a precedent that Apple will soon follow with their iOS App Store.
```

Comment:
This is a big part for me - not just the how, but the why.

Make a big deal of this in your lead in and perhaps expand a little on why this is awesome.

Why is it better than building native apps?  etc.